### ISSUE TITLE
Option to disable modesetting

### ISSUE THREAD

**Issue by StephanX** [hertg/egpu-switcher#95]
Hi!  

On my setup, everything works great, except I need all of `Section "Module"` removed for the xorg config file (for the egpu) to work.  I'd love some sort of flag like `'--nomodesetting'` so that only the `Device` group is generated.

When modesetting is activated, gdm does not land on any connected monitor.  

egpu-switcher version 0.18.1 on Arch (up to date) via Arch/Aur (gnome if that helps)
i7-6700HQ Alienware r13 + igpu gtx1060 + evga gtx 1070 / Akitio Node

**Comment by hertg** [comment]
@timpauli Good to know, thanks for the report. I'll prioritize this issue higher then.

Could you reproduce the issue again and upload the logs for the failed X.Org session? Just so I get some background information on what exactly is happening there.

```sh
cat /var/log/Xorg.0.log
```

---

A potential workaround might be to enable [early KMS start](https://wiki.archlinux.org/title/Kernel_mode_setting#Early_KMS_start)?

**Comment by timpauli** [comment]
`/var/lib/gdm3/.local/share/xorg/Xorg.0.log`

[Xorg.0.log](https://github.com/hertg/egpu-switcher/files/10885647/Xorg.0.log)

Did not try the early KMS start yet.

Thank you

**Comment by hertg** [comment]
@StephanX @timpauli 

I implemented this in [0.19.0](https://github.com/hertg/egpu-switcher/releases/tag/0.19.0). The switch command now has the `--nomodesetting` flag. Or for an even easier way, update your config so that the modesetting module is always omitted.

*/etc/egpu-switcher/config.yaml*
```yaml
egpu:
    nomodesetting: true # add this line
    driver: amdgpu
    id: 1153611719250962689
```

**Comment by ger5ones** [comment]
@hertg 
Hi.
It didn't work for me.
Would you have additional suggestions, please?
I have an egpu build that will only work on linux (having BSOD in windows) so I'm relying on this.
Thank you
Regards
